LibreNMS/Alert/AlertUtil.php

```diff
@@ -44,9 +48,7 @@ class AlertUtil
      */
     private static function getRuleId($alert_id)
```

Comment:
Hmm. Can’t we get rid of this function and just call alert find directly where it’s used ?

Reply:
And for most other functions in this class